helm/cert-manager-app/values.schema.json

```diff
@@ -103,6 +103,9 @@
         "crds": {
             "type": "object",
             "properties": {
+                "backoffLimit": {
+                    "type": "integer"
```

Comment:
Maybe also allow `null`? But no strong opinion on this.

Reply:
no that won't work - if you don't actually set the field then the rendered manifest will be invalid. `null` values in the schema only make sense when there is some logic in the chart which does something _if_ the value is null. in this case the template needs the value to be an int - null will break rendering.

Reply:
`null` is special and means for helm "[delete that key](https://docs.helm.sh/docs/chart_template_guide/values_files/#deleting-a-default-key)". This was actually important for changing a default PodDisruptionBudget from `minAvailable` to `maxUnavailable` since only one of them is allowed and the other needs to be deleted. Setting `null` here would then mean, leave out this field and use cluster default. But maybe not really important.

Reply:
oh interesting - i wasn't aware of that. sure, not that important here but definitely useful to know about!